.gitmodules

```diff
@@ -7,15 +7,9 @@
 [submodule "cosmos-proto"]
 	path = cosmos-proto
 	url = https://github.com/cosmos/cosmos-proto.git
-[submodule "cosmos-sdk"]
-	path = cosmos-sdk
-	url = https://github.com/cosmos/cosmos-sdk.git
 [submodule "xpla"]
 	path = xpla
 	url = https://github.com/xpladev/xpla.git
-[submodule "ethermint"]
-	path = ethermint
-	url = https://github.com/xpladev/ethermint.git
```

Comment:
It would be nice to keep ethermint for backwards compatibility (pre-upgrade tx), etc. If not, it would be a good idea to create legacy/ethermint/proto in xpladev/xpla.